components/script/webdriver_handlers.rs

```diff
@@ -1922,24 +1922,19 @@ fn get_element_pointer_interactable_paint_tree(
     document: &Document,
     can_gc: CanGc,
 ) -> Vec<DomRoot<Element>> {
-    // Step 2. Let rectangles be the DOMRect sequence returned by calling getClientRects()
-    let rect = element.GetClientRects(can_gc);
-
-    if rect.first().is_some() {
-        // Step 4. Let center point be the in-view center point of
-        // the first indexed element in rectangles.
-        match get_element_in_view_center_point(element, can_gc) {
-            // Step 5. Return the elements from point given the coordinates center point
-            Some(center_point) => document.ElementsFromPoint(
-                Finite::wrap(center_point.x as f64),
-                Finite::wrap(center_point.y as f64),
-            ),
-            None => Vec::new(),
-        }
-    } else {
-        // Step 3. If rectangles has the length of 0, return an empty sequence
-        Vec::new()
+    // Step 1. If element is not in the same tree as session's
+    // current browsing context's active document, return an empty sequence.
+    if !element.is_connected() {
+        return Vec::new();
     }
+
+    // Step 2 - 5: Spec has bugs in description. Will add description after changing spec.
```

Comment:
Could you change the comment into what these steps are supposed to do, with a reference to the spec bug?